### PR TITLE
doc: document inactive vaults

### PR DESCRIPTION
**Pull request by darosior**

### CODE REVIEW

File: doc/API.md [hunk 1]
```diff
@@ -32,26 +32,31 @@ Display general information about the current daemon state.
...
 ### Vault resource
 
-| Field    | Type   | Description                                             |
-| -------- | ------ | ------------------------------------------------------- |
-| `amount` | int    | Amount of the vault in satoshis                         |
-| `txid`   | string | Unique ID of the vault deposit transaction              |
-| `status` | string | Status of the vault ([vault statuses](#vault-statuses)) |
+| Field         | Type   | Description                                                                          |
+| ------------- | ------ | ------------------------------------------------------------------------------------ |
+| `blockheight` | int    | Block height at which the vault deposit transaction was confirmed (0 if unconfirmed) |
```
Comment (edouardparis) [inline doc/API.md:51]: can you keep the table sorted alphabetically please ?

Comment (darosior) [reply doc/API.md:51]: Hmm so *this* was the rationale for this order!

Comment (darosior) [reply doc/API.md:52]: Done

**Comment by edouardparis** [comment]
ACK e9c9b7f30bb1b6599af74d701e97bf4b023fa400

**Pull request merged**